perfzero/lib/perfzero/utils.py

```diff
@@ -424,26 +424,23 @@ def setup_tpu(parameters):
     True if an error occurs during setup.
   """
   try:
-    _install_tpu_tool()
-
+    base_cmd = 'gcloud compute tpus execution-groups create'
     args = [
+        '--tpu-only',
         '--name={}'.format(parameters.get('name')),
         '--project={}'.format(parameters.get('project')),
         '--zone={}'.format(parameters.get('zone')),
-        '--tpu-size={}'.format(parameters.get('size')),
+        '--accelerator-type={}'.format(parameters.get('size')),
```

Comment:
Is `accelerator-type (v3-32)` in the `size` attribute?

Reply:
Yes - it was the past setup.